templates/layouts/main.html

```diff
@@ -61,8 +61,7 @@
   <header class="site-header">
     <div id="notice" aria-hidden="true" class="section--blue">
       <a id="notice-close" class="js-action js-hide link--icon" tabindex="0" data-hides="notice"><i class="ti-close"></i></a>
-      <h3>This is a work in progress</h3>
-      <p>This site is in <a href="https://18f.gsa.gov/dashboard/stages/#beta" target="_blank">beta</a>. <a href="/#progress">Find out what’s here and what’s missing</a>.</p>
+      <h3>This is a work in progress... this site is in <a href="https://18f.gsa.gov/dashboard/stages/#beta" target="_blank">beta</a>. <a href="/#progress">Find out what’s here and what’s missing</a>.</h3>
```

Comment:
Can we ditch the ellipsis? Seems too informal. 

Also, h3 feels real big. How about h4?